pyof/v0x04/controller2switch/common.py

```diff
@@ -480,7 +482,19 @@ def unpack(self, buff=None, offset=0):
 
     def update_length(self):
         """Update the length of current instance."""
-        self.length = self.get_size()
+        self.length = super().get_size()
```

Comment:
<img width="911" height="448" alt="Image" src="https://github.com/user-attachments/assets/e1870352-6108-43b0-9e10-b8a153897d55" />

I've just realized this class method `unpack()` hasn't been covered yet. Good to cover this in the future.

Reply:
Great. Appreciated @italovalcy